src/Components/Details/Details.css

```diff
@@ -0,0 +1,68 @@
+.details__details-view {
+  width: 100vw;
```

Comment:
Don't forget to order your CSS properties in some consistent manner within each rule.